ui/components/app/signature-request/signature-request.component.js

```diff
@@ -2,7 +2,7 @@ import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
```

Comment:
This is place in code where we use this component for v3, v4 requests https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/confirm-transaction/conf-tx.js#L121